addon/-private/array.ts

```diff
@@ -3,36 +3,36 @@ import {
   consumeTag,
   dirtyTag,
   consumeCollection,
-  dirtyCollection
-} from 'tracked-maps-and-sets/-private/util';
+  dirtyCollection,
+} from "tracked-maps-and-sets/-private/util";
```

Comment:
Converting to double quotes seems strange, was this a prettier setting?

Reply:
Doh, yes, auto-fix from VS Code. Thought I'd avoided all of these. Will update. Thanks!